### PR TITLE
UPBGE: Fix not culled texts in sintel files

### PR DESCRIPTION
**Pull request by youle31**

### CODE REVIEW

File: source/gameengine/Rasterizer/RAS_OpenGLRasterizer/RAS_OpenGLRasterizer.cpp [hunk 1]
```diff
@@ -2134,6 +2134,9 @@ void RAS_OpenGLRasterizer::RenderText3D(
 	/* gl prepping */
 	DisableForText();
 
+	Enable(RAS_CULL_FACE);
+	glCullFace(GL_BACK);
```
Comment (panzergame) [inline source/gameengine/Rasterizer/RAS_OpenGLRasterizer/RAS_OpenGLRasterizer.cpp]: This call to different things than SetFrontFace(true) ?

**Comment by panzergame** [comment]
I think the call to Enable(RAS_CULL_FACE) can be moved in DisableForText if no issues are encountered.
Thanks for the fix.

**Comment by youle31** [comment]
ok, done :)

**Comment by youle31** [comment]
I think it's ok to merge. The only think is that backface culling is always enabled. Maybe later we could link the material backface culling option in material tab to have the choice to enable/disable it, but for now I think it's ok...

**Pull request closed without merging**